cookies/accession/zotero.py

```diff
@@ -430,14 +429,10 @@ def __iter__(self):
     def next(self):
         next_entry = None
         while next_entry is None:
-            try:
-                next_entry = self.current_entries.next()
-            except (StopIteration, AttributeError):
-                try:
-                    self.current_class = self.classes.pop()
-                except IndexError:    # Out of classes.
-                    raise StopIteration()
-                self.current_entries = self._get_resources_nodes(self.current_class)
+            next_entry = self.current_entries.next()
+            if str(next(self.graph.objects(subject=next_entry, predicate=ZOTERO.itemType))) == \
```

Comment:
what if 'self.graph.objects(subject=next_entry, predicate=ZOTERO.itemType)' is None?